website-frontend/src/components/Home.js

```diff
@@ -1,10 +1,28 @@
 // @nhomnhomm Complete this
+import cover from '../images/cover.png'
+import decoration from '../images/decoration.png'
+
+const Hero = () => {
+    return (
+        <div className="container" id="hero">
+            <div className="row h-100 align-items-center">
+                <div className="col-5">
+                    <h1 className="text-secondary">Fulbright <br></br> STEM Club.</h1>
```

Comment:
What's the `<br>` for?
Also, I guess this text should be in bold style.

Reply:
`<br>` is for breaking the Fulbright STEM Club line accordingly to the Figma design ạ. 
And yes, I have added the bold style, thank you for reminding me anh.